meson.build

```diff
@@ -1887,6 +1890,16 @@ if conf.get('ENABLE_TIMESYNCD') == 1
                    install_dir : rootlibexecdir)
 endif
 
+if conf.get('ENABLE_TIMESYNCWAITD') == 1
+        executable('systemd-timesyncwaitd',
```

Comment:
Would you take "systemd-time-wait-synchronized.service"?  This really has nothing to do with timesyncd except that it exists because timesyncd can't satisfy the expectations of "time-sync.target" without an external helper.  It's just as useful for any service that attempts to synchronize time (ntp, htp, home-grown GPS processors, ...).

Reply:
And "timewaitsync" for the application?